source/guide/3.2/plugin-hostpool.md

```diff
@@ -0,0 +1,143 @@
+  * `cloudify.interfaces.worker_installer.start` starts the agent worker.
+  * `cloudify.interfaces.worker_installer.stop` stops the agent worker.
+  * `cloudify.interfaces.worker_installer.uninstall` uninstalls the agent worker.
+  * `cloudify.interfaces.worker_installer.restart` restarts the agent worker.
+
+## cloudify.hostpool.nodes.WindowsHost
+
+**Derived From:** [cloudify.openstack.nodes.Compute](#cloudifyopenstackserver)
+
+Windows host type has the same mapped operations as default one but uses windows agent worker installer. Use this type when working with a Windows node.
+
+
+## Runtime Properties
+
+Node instances of any of the types defined in this plugin get set with the following runtime properties during the `cloudify.interfaces.lifecycle.start` operation:
+
+  * `ip` the private IP (ip on the internal network) of the server.
+  * `cloudify_agent` node's authentication information, as retrieved from the Host Pool service:
+    * `user` the administrator username.
+    * `key` the keyfile for the administrator user.
+    * `password` the password for the administrator user. 
+    * `port` port of the available connection.
+
+# Example
+
+This example will show how to use most of the types in this plugin on the [Node Cellar example](https://github.com/cloudify-cosmo/cloudify-nodecellar-example).
+
+{% togglecloak id=1 %}
+Example
+{% endtogglecloak %}
+
+{% gcloak 1 %}
+
+{% highlight yaml %}
+
+inputs:
+  hostpool_service_url:
+      type: string
+      description: >
+        The url of the host pool service to acquire and release nodes.
+
+node_templates:
+
+  mongod_host:
+    type: cloudify.hostpool.nodes.Host
+    interfaces:
+      cloudify.interfaces.lifecycle:
+          start:
+            inputs:
+              service_url: { get_input: hostpool_service_url }
+          delete:
+            inputs:
+              service_url: { get_input: hostpool_service_url }
+
+  mongod:
+    type: nodecellar.nodes.MongoDatabase
+    properties:
+      port: 27017
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        configure: scripts/mongo/install-pymongo.sh
+    relationships:
+      - type: cloudify.relationships.contained_in
+        target: mongod_host
+
+  nodejs_host:
+    type: cloudify.hostpool.nodes.Host
+    interfaces:
+      cloudify.interfaces.lifecycle:
+          start:
+            inputs:
+              service_url: { get_input: hostpool_service_url }
+          delete:
+            inputs:
+              service_url: { get_input: hostpool_service_url }
+
+  nodejs:
+    type: nodecellar.nodes.NodeJSServer
+    relationships:
+      - type: cloudify.relationships.contained_in
+        target: nodejs_host
+
+  nodecellar:
+    type: nodecellar.nodes.NodecellarApplicationModule
+    properties:
+      port: 8080
+    relationships:
+      - type: node_connected_to_mongo
+        target: mongod
+      - type: node_contained_in_nodejs
+        target: nodejs
+{%endhighlight%}
+
+Node by node explanation:
```

Comment:
i think you should present a more trivial example here. just one host one node. and say that to view a more detailed example you can look at the new nodecellar example in the nodecellar example repository